docker/webui/Dockerfile

```diff
@@ -18,7 +18,6 @@ RUN gensslcert && \
     a2enmod ssl && \
     a2enmod rewrite && \
     a2enflag SSL
-ADD openqa-ssl.conf /etc/apache2/vhosts.d/openqa-ssl.conf
```

Comment:
I don't think we want to change this by default. Keeping SSL enabled is still a good idea and we also configure the apache further up accordingly.

Reply:
The problem is that the certificates that refers this configuration doesn't exists. maybe a good option could be do this optional parametrizing the inclusion of this configuration and their certificates on the docker-compose. But I think we shouldn't include in the image some configuration that breaks the basic execution. I could prepare this, or could be documented to show how to do.